### ISSUE TITLE
Can't log on addon after a reboot

### ISSUE THREAD

**Issue by fabienmaquin** [iMicknl/ha-tahoma#832]
### Did you read the instructions?

- [X] I have read the [README](https://github.com/iMicknl/ha-tahoma/blob/master/README.md), including the [advanced debugging](https://github.com/iMicknl/ha-tahoma/blob/master/README.md#advanced) section.

### The problem

Got the following error after a reboot:
`Translation Error: The intl string context variable "gateway_id" was not provided to the string "Passerelle : {gateway_id}"`

Can't log on Cozytouch after it was finally working well this afternoon.... dang !

![image](https://user-images.githubusercontent.com/76770318/205143667-70449338-5d78-406c-8102-d4a84d40d5a5.png)

### What version of this integration (ha-tahoma) has the issue?

2.14.0

### What version of Home Assistant Core has the issue?

2022.12.0b0

### Which gateway / hub do you use?

Cozytouch Sauter/atlantic

### Home Assistant log

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks. The first log line must start with "Added device (Device...".
```

</details>

**Comment by iMicknl** [comment]
I am not sure what you try to achieve? If you want to use the new CozyTouch login implementation, follow the previous instructions.

1. Delete the custom component (ha-Tahoma)
2. Use the Overkiz integration from core

On the screenshot you are using ha-tahoma, which doesn't support the new CozyTouch login as discussed before..

**Comment by fabienmaquin** [comment]
@iMicknl Actually if use the Overkiz Integration from core I don't have the full preset (only comfort), yet if I Use the custom one I well get Comfort -2, Comfort -1 and Comfort.

Plus, when I got the login issue, i can't log on both of the addons. (Custom and Core one) 

I got the feeling that when i reboot i'm loosing connexion and the only i found out to get it back was to full uninstall addon, reboot, wait for 24h and it will probably come back. But i'm loosing my heater for 24h not really cool...

**Comment by fabienmaquin** [comment]
And i can confirm that the custom component support the CozyTouch login by the way. (when it works ^^)

**Comment by iMicknl** [comment]
@fabienmaquin CozyTouch login should be fixed in the public beta, or in 2022.12 when it is released. Did you try this?

Please create a new issue on https://github.com/home-assistant/core/issues if you face any issues with the core integration. (like missing presets)
